app/pods/components/publications/publication/activity-request-panel/component.js

```diff
@@ -2,9 +2,23 @@ import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 
+/*
+*  @activity={{activity}}
+*  @finishedToggle: function
+*  @cancelExisting: function
+*  @translation: true/false
+ */
+
 export default class ActivityRequestPanel extends Component {
   @tracked isCollapsed = false;
 
+  get isTranslation() {
+    if (this.args.translation === true) {
+      return true;
+    }
+    return false;
```

Comment:
Deze code lijkt me een beetje vuil.. om het iets properder te maken kan je het misschien `modus` noemen en checken t.o.v. een generieke modus ipv een rechtstreekse implementatie van isVertaling. Stel je voor dat handtekenen een gelijkaardige view heeft, dan zou je hier je logica al moeten aanpassen omdat je jezelf zo gelimiteerd hebt in keuzes.